src/PlusDataCollection/OptiTrack/vtkPlusOptiTrack.cxx

```diff
@@ -124,6 +126,22 @@ PlusStatus vtkPlusOptiTrack::ReadConfiguration(vtkXMLDataElement* rootConfigElem
   XML_READ_STRING_ATTRIBUTE_NONMEMBER_REQUIRED(ProjectFile, this->Internal->ProjectFile, deviceConfig);
   XML_READ_BOOL_ATTRIBUTE_NONMEMBER_REQUIRED(RunMotiveInBackground, this->Internal->RunMotiveInBackground, deviceConfig);
 
+  if (this->Internal->RunMotiveInBackground)
+  {
+    // Motive is running in background, tools cannot be modified therefore never udpate tools
+    this->Internal->UpdateTools = false;
+  }
+  else
+  {
+    this->Internal->UpdateTools = true;
```

Comment:
It may be simpler to merge UpdateTools flag and ToolUpdateRate variables into a single numeric variable. If rate is negative then it could refer to never update, if positive then it would update at the specified rate.